include/symbolic_functions.h

```diff
@@ -3,30 +3,30 @@
 
 namespace amici {
 
-double amilog(double x);
+double log(double x);
 double dirac(double x);
 double heaviside(double x);
 
-double am_min(double a, double b, double c);
-double Dam_min(int id, double a, double b, double c);
-double am_max(double a, double b, double c);
-double Dam_max(int id, double a, double b, double c);
+double min(double a, double b, double c);
+double Dmin(int id, double a, double b, double c);
+double max(double a, double b, double c);
+double Dmax(int id, double a, double b, double c);
 
-int amiIsNaN(double what);
-int amiIsInf(double what);
-double amiGetNaN();
+int isNaN(double what);
+int isInf(double what);
+double getNaN();
 
 /* sign */
 double sign(double x);
 
 /* splines */
 
-double am_spline(double t, int num, ...);
-double am_spline_pos(double t, int num, ...);
-double am_Dspline(int id, double t, int num, ...);
-double am_Dspline_pos(int id, double t, int num, ...);
-double am_DDspline(int id1, int id2, double t, int num, ...);
-double am_DDspline_pos(int id1, int id2, double t, int num, ...);
+double spline(double t, int num, ...);
+double spline_pos(double t, int num, ...);
+double Dspline(int id, double t, int num, ...);
+double Dspline_pos(int id, double t, int num, ...);
+double DDspline(int id1, int id2, double t, int num, ...);
+double DDspline_pos(int id1, int id2, double t, int num, ...);
```

Comment:
All above: lower-case initial would be more conventional

Reply:
true, but the regexp expression in gccode.m would be more of a pain